pyproject.toml

```diff
@@ -204,6 +205,25 @@ ignore = [
 [tool.ruff.lint.pydocstyle]
 convention = "pep257" # https://docs.astral.sh/ruff/settings/#lint_pydocstyle_convention
 
+[tool.ruff.lint.flake8-import-conventions.aliases]
+# Same as default
+"altair"="alt"
+"matplotlib"="mpl"
+"matplotlib.pyplot"="plt"
+"numpy"="np"
+"pandas"="pd"
+"seaborn"="sns"
+"tensorflow"="tf"
+# "tkinter"="tk" # Causes name conflict in stdlib/tkinter/ttk.py with Style.tk
```

Comment:
I wonder if you could still use `extend-aliases` and set `"tkinter"="tkinter"` to disable that one.

Reply:
That actually works! Even if we don't end up using these rules, I opened a request to document that behaviour: https://github.com/astral-sh/ruff/issues/17097